README.md

```diff
@@ -6,13 +6,19 @@ This is a microservice that allows user to create comments, edit comments and al
 
 For quick navigation:
 
+- [How it Works](#how-it-works)
 - [Getting Started](#getting-started)
 - [Linting Your Code](#linting-your-code)
 - [Testing Your Code](#testing-your-code)
 - [Contributing Your Code](#contributing-your-code)
 - [Endpoints Documentation](#endpoints-documentation)
 - [Schema Design Explanation](#schema-design-explanation)
 
+
+## <a name="how-it-works"></a> How it works
```

Comment:
This whole section is empty so I don't see why it was added.

Reply:
Its a placeholder